### ISSUE TITLE
Desktop Zoom (Super & + keys) not working in Deepin v20

### ISSUE THREAD

**Issue by mmohsinkhan** [linuxdeepin/developer-center#1824]
I am a software developer with eye sight problem, I need screen magnification to use my computer and carry out routine work.

I have been using Deepin 15 as my daily driver for almost a year. Screen magnification works fine on Deepin 15 but now it doesn't seems to work on Deepin v20.
Keyboard shortcuts working on Deepin 15
Zoom In: Super & + keys
Zoom Out: Super & - keys

I want to continue using Deepin and upgrade to v20 but without screen magnification it won't be possible.
Please re-enable the feature in v20.

Best Regards

**Comment by xmuli** [comment]
Hello, depin v20 beta has `crtl + +` and `crtl + -` for scaling; this feature is always present; you can install the v20 environment on another computer and find that it is present!

**Comment by xmuli** [comment]
For v15, it may not be possible to upgrade to v20 via repository, but it may be more suitable to reinstall it (we suggest to backup your data beforehand, for v20 full version via repository, we are still testing it internally, of course, you can also wait for the later standard version of v20); Best wishes to you!

**Comment by mmohsinkhan** [comment]
Great. Thanks a lot.
I will test it on a VM.

**Comment by mmohsinkhan** [comment]
@xmuli I tried the mentioned shortcut on a Deepin v20 Beta VM installation but it did not work.
Crtl + + increases font size in terminal and in browser but does not turn on screen magnification. Please guide.

**Comment by xmuli** [comment]
Try `win + +`, and `win + -` to enlarge the desktop

**Comment by mmohsinkhan** [comment]
> Try `win + +`, and `win + -` to enlarge the desktop

Yes I did this using "Win + +", actually by " (Super & +)" I meant "Win + +".
Problem is that it is not usable anymore. Zoom does not follow mouse pointer (like it used to in Deepin v15) and frame drop is quite obvious.

If you have a working Deepin v20 Beta installation (not VM), you can verify it yourself.
Thanks for you support.

**Comment by mmohsinkhan** [comment]
Hi,

Is there any plan to fix this issue ? I want to use Deepin v20 but this limitation is stopping me.

Best Regards

**Comment by xmuli** [comment]
Hello, actually I always remember you, thank you for your love of deepin

Just now, I have communicated with the operation manager and the product. They have paid attention to this and will fix this problem later

**Comment by mmohsinkhan** [comment]
Thanks. I am extremely grateful.

**Comment by xmuli** [comment]
happy   :smiley: :smiley:

**Comment by mmohsinkhan** [comment]
Will be 😀 When I would be able to use v20.
But my thanks for your support.

**Comment by mmohsinkhan** [comment]
Want to share some information that I think might help in figuring out the component causing this problem.

Because I want Deepin so badly, I installed EndeavourOS (Arch based) as if offers Deepin v20 Beta DDE as a desktop option :) .
Surprisingly desktop zoom was working completely fine on DDE 20 with EndeavourOS, so I suspect this problem has to do with base system or maybe some sort of driver configuration.

Thanks

**Comment by xmuli** [comment]
For the time being, you can use this function normally, I am very happy. Maybe @felixonmars  knows the reason for this, (Why does dde move with the mouse in arch is normal)
